$NetBSD: patch-src_3rdparty_chromium_native__client__sdk_src_BUILD.gn,v 1.1 2025/12/21 09:38:33 markd Exp $

* Based on OpenBSD's qt6-qtwebengine patches, and
  pkgsrc's qt5-qtwebengine patches

--- src/3rdparty/chromium/native_client_sdk/src/BUILD.gn.orig	2025-05-29 01:27:28.000000000 +0000
+++ src/3rdparty/chromium/native_client_sdk/src/BUILD.gn
@@ -52,7 +52,7 @@ if (enable_nacl) {
 
     # TODO(binji): dump_syms doesn't currently build on Windows. See
     # http://crbug.com/245456
-    if (!is_win && target_cpu == "x64") {
+    if (!is_win && target_cpu == "x64" && !is_bsd) {
       deps += [
         "//third_party/breakpad:dump_syms($host_system_allocator_toolchain)",
         "//third_party/breakpad:minidump_dump($host_system_allocator_toolchain)",
